### PR TITLE
Protect Yourself Animation + Styling

### PR DESCRIPTION
**Pull request by luigi-rosso**
- [x] Provided detailed pull request description and a succinct title (consider template below for guidance).
- [x] Tested your changes, especially after any code review iterations.
- [x] Included any relevant screenshots of UI updates.
- [x] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md).
- [x] Verified all contributions are properly licensed pursuant to the [LICENSE](https://github.com/WorldHealthOrganization/app/blob/master/LICENSE) file in the root of the repository.
- [x] Verified your name is in the [content/credits.yaml](https://github.com/WorldHealthOrganization/app/blob/master/content/credits.yaml) file (if you want it to be).

## What does this PR accomplish?
Adds preliminary animations to the Protect Yourself section. We are missing the distancing one which we are wrapping up tonight. We should have it ready Sunday. We also added styling to the Protect Yourself section matching the latest designs. Addresses #520

## Did you add any dependencies?
Yes we added the following dependencies:
```
  visibility_detector: ^0.1.4
  flare_flutter: ^2.0.1
  path_drawing: ^0.4.1
```
**visibility_detector** - to help qualify if a widget is on/off screen to optimize out those animations, helps low end devices not have to play a bunch of unnecessary animations
**flare_flutter** - support for Rive animations in Flutter
**path_drawing** - support for SVG path parsing so we can match the back arrow from the Figma designs

## How did you test the change?

Ran on a few different devices and emulator/simulators.
rive_animations

### CODE REVIEW

**Comment by luigi-rosso** [comment]
We ran into some issues with the internationalization package not behaving as documented so we weren't able to template text strings, but we can work with whoever set that up for this project to get it fixed up.

**Comment by advayDev1** [comment]
This is awesome!  Thank you so much!  @luigi-rosso 
Do we really need the fonts?  
* The animations are less than 100kb, but the fonts add ~2MB.
* Are these Apple SF fonts really MIT licensed?

File: client/flutter/ios/Podfile.lock [hunk 1]
```diff
@@ -55,4 +55,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 083258d7f5e80b42ea9bfee905fe93049bc04c64
 
-COCOAPODS: 1.9.1
+COCOAPODS: 1.8.4
```
Comment (advayDev1) [inline client/flutter/ios/Podfile.lock]: Please use the latest cocoapods
```suggestion
COCOAPODS: 1.9.1
```

**Comment by advayDev1** [comment]
Dependencies checked:
- visibility_detector / official flutter.dev
- flare_flutter / official rive
- path_drawing / by @dnfield , google flutter end

**Comment by luigi-rosso** [comment]
Someone with better knowledge of Apple's font license should review the SF Fonts, they're definitely permitted on an iOS app, not sure if it'll cause problems on the Android build. We added them as they were the fonts used in the Figma design files we were given as a guideline. Happy to change as required, or have another quick sync with the design team if they need to change.

**Comment by brunobowden** [comment]
Removed from tree history due to ttf licensing issues. Please create a new PR for this. Engineering wise, you have to be very careful to change a dependency to an older version (e.g. Cocoapods), that should almost never happen. Also, is it possible to get bigger movie of animations to see them up close. It would be good to have design approve this too.
@patniemeyer @luigi-rosso @advayDev1

**Comment by advayDev1** [comment]
Thanks Bruno!

**Pull request merged**